### PR TITLE
Simput fixes

### PR DESCRIPTION
**Pull request by TristanWright**
fixes: 
- #573 
- #577

### CODE REVIEW

**Comment by cjh1** [comment]
@TristanWright is there something special I need to do to deploy this change, I am not seeing it fix #577

File: src/workflows/pyfr/components/steps/Input/index.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const SimputPanel = React.createClass({
   componentWillMount() {
     var iniFile = this.props.simulation.metadata.inputFolder.files.ini;
     var jsonData = this.props.simulation.steps[this.props.step].metadata.model;
-
+    console.log(iniFile, '\n', jsonData);
```
Comment (jourdain) [inline src/workflows/pyfr/components/steps/Input/index.js]: Do you want to keep debug output?

File: src/workflows/pyfr/components/steps/Input/index.js [hunk 2]
```diff
@@ -178,12 +182,12 @@ const SimputPanel = React.createClass({
 
     // Update ini file content
     try {
+      delete jsonData.data.backend; // we want to use the backend configured from the cluster
+      const convertedData = this.props.convert(jsonData);
+      console.log(convertedData);
```
Comment (jourdain) [inline src/workflows/pyfr/components/steps/Input/index.js]: debug output?

File: src/workflows/pyfr/components/steps/Input/index.js [hunk 3]
```diff
@@ -178,12 +182,12 @@ const SimputPanel = React.createClass({
...
+      const content = convertedData.results['pyfr.ini'];
+      console.log('try to save content', content.length);
```
Comment (jourdain) [inline src/workflows/pyfr/components/steps/Input/index.js]: debug output?

File: src/workflows/pyfr/components/steps/Input/index.js [hunk 4]
```diff
@@ -218,10 +239,12 @@ const SimputPanel = React.createClass({
     const data = this.state.viewData,
       keypath = newData.id.split('.'),
       attrName = keypath.shift();
-    console.log(attrName, keypath, newData);
     data[attrName][keypath.join('.')].value = newData.value;
-    console.log(data, data[attrName][keypath.join('.')].value);
     this.setState({ viewData: data });
+    if (process.env.NODE_ENV !== 'production') {
```
Comment (jourdain) [inline src/workflows/pyfr/components/steps/Input/index.js]: Do we want to keep that output?

**Comment by jourdain** [comment]
I'm not sure to know how that can fix #577 though.

**Comment by TristanWright** [comment]
> I'm not sure to know how that can fix #577 though.
correct, sorry it fixes #576

**Comment by jourdain** [comment]
Let's merge it then

**Pull request merged**